<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotSettingsDialog</class>
 <widget class="QDialog" name="PlotSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>663</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>8</number>
     </property>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>8</number>
         </property>
         <property name="margin">
          <number>10</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="8" column="2">
            <widget class="QCheckBox" name="IsShowYscaleNum">
             <property name="text">
              <string>Display Zoom Factor</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_YRange">
             <item>
              <widget class="QDoubleSpinBox" name="doubleSpinBox_YRangeMin">
               <property name="decimals">
                <number>4</number>
               </property>
               <property name="minimum">
                <double>-99999999.000000000000000</double>
               </property>
               <property name="maximum">
                <double>99999999.000000000000000</double>
               </property>
               <property name="value">
                <double>0.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_YRangeTo">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="doubleSpinBox_YRangeMax">
               <property name="decimals">
                <number>4</number>
               </property>
               <property name="minimum">
                <double>-99999999.000000000000000</double>
               </property>
               <property name="maximum">
                <double>99999999.000000000000000</double>
               </property>
               <property name="value">
                <double>0.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="label_YRange">
             <property name="text">
              <string>Y-Axis Range</string>
             </property>
            </widget>
           </item>
           <item row="9" column="2">
            <widget class="QCheckBox" name="checkBox_AutoYAxis">
             <property name="text">
              <string>Dynamic Range</string>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="label_YDecimal">
             <property name="text">
              <string>Y-Axis Decimal Places</string>
             </property>
            </widget>
           </item>
           <item row="12" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Background Color</string>
             </property>
            </widget>
           </item>
           <item row="13" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Mark Location</string>
             </property>
            </widget>
           </item>
           <item row="14" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Grid Type</string>
             </property>
            </widget>
           </item>
           <item row="15" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Data Save Mode</string>
             </property>
            </widget>
           </item>
           <item row="7" column="2">
            <widget class="QCheckBox" name="checkBox_ShowYTitle">
             <property name="text">
              <string>Display Y-Axis Label</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QLineEdit" name="lineEdit_YTitle">
             <property name="maxLength">
              <number>64</number>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Y-Axis Label</string>
             </property>
            </widget>
           </item>
           <item row="6" column="2">
            <widget class="QCheckBox" name="checkBox_xBottomLog">
             <property name="text">
              <string>Logarithmic Form</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Y-Axis Scaling Factor</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QComboBox" name="YscaleCombo"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>X-Axis Label</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_XType">
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>X-Axis Type</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_XRange2">
             <property name="text">
              <string>X-Axis Range</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEdit_XTitle">
             <property name="maxLength">
              <number>64</number>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QCheckBox" name="checkBox_ShowSubTitle">
             <property name="text">
              <string>Show Secondary Label</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QCheckBox" name="checkBox_ShowXTitle">
             <property name="text">
              <string>Show X-Axis Label</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_XRange1">
             <property name="text">
              <string>X-Axis Length</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_XDecimal">
             <property name="maximumSize">
              <size>
               <width>228</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>X-Axis Decimal Places</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_XRange">
             <item>
              <widget class="QDoubleSpinBox" name="doubleSpinBox_XRangeMin">
               <property name="decimals">
                <number>4</number>
               </property>
               <property name="minimum">
                <double>-99999999.000000000000000</double>
               </property>
               <property name="maximum">
                <double>99999999.000000000000000</double>
               </property>
               <property name="value">
                <double>0.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_XRangeTo">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="doubleSpinBox_XRangeMax">
               <property name="decimals">
                <number>4</number>
               </property>
               <property name="minimum">
                <double>-99999999.000000000000000</double>
               </property>
               <property name="maximum">
                <double>99999999.000000000000000</double>
               </property>
               <property name="value">
                <double>0.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Main Label</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_Title">
             <property name="maxLength">
              <number>64</number>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QCheckBox" name="checkBox_ShowTitle">
             <property name="text">
              <string>Show Main Label</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Sub Label</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_SubTitle">
             <property name="maxLength">
              <number>64</number>
             </property>
            </widget>
           </item>
           <item row="3" column="1" colspan="2">
            <widget class="QComboBox" name="comboBox_XType">
             <item>
              <property name="text">
               <string>Point Number</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Time Stamp</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Fixed Range</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="4" column="1" colspan="2">
            <widget class="QDoubleSpinBox" name="doubleSpinBox_XRange">
             <property name="decimals">
              <number>4</number>
             </property>
             <property name="minimum">
              <double>0.000100000000000</double>
             </property>
             <property name="maximum">
              <double>10000.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="5" column="1" colspan="2">
            <widget class="QComboBox" name="comboBox_XDecimal">
             <item>
              <property name="text">
               <string>Auto</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>0</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>7</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="10" column="1" colspan="2">
            <widget class="QComboBox" name="comboBox_YDecimal">
             <item>
              <property name="text">
               <string>Auto</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>0</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>7</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="13" column="1" colspan="2">
            <widget class="QComboBox" name="comboBox_LegendSide">
             <item>
              <property name="text">
               <string>Left</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Right</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Up</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Down</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="14" column="1" colspan="2">
            <widget class="QComboBox" name="comboBox_GridType">
             <item>
              <property name="text">
               <string>None</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Dotted Line</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="15" column="1" colspan="2">
            <widget class="QComboBox" name="comboBox_SaveDataMode">
             <item>
              <property name="text">
               <string>Only save data within the current X-Axis range</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Save all data</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="12" column="2">
            <widget class="QPushButton" name="colorBtn">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>Setting</string>
             </property>
            </widget>
           </item>
           <item row="12" column="1">
            <widget class="QLineEdit" name="bgcolor_edit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="var">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Variable</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_Delete">
             <property name="minimumSize">
              <size>
               <width>128</width>
               <height>0</height>
              </size>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>Delete Variable</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_Clear">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>Clear</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="CurveSettingTreeView" name="treeView"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CurveSettingTreeView</class>
   <extends>QTreeView</extends>
   <header>curvesettingtreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
